go.mod

```diff
@@ -97,25 +94,25 @@ require (
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.11.0
 	github.com/go-resty/resty/v2 v2.13.1
-	github.com/go-webauthn/webauthn v0.10.2
+	github.com/go-webauthn/webauthn v0.11.0
```

Comment:
Breaking changes galore, I'll see about this update in a PR of its own.

- https://github.com/go-webauthn/webauthn/releases/tag/v0.11.0